packages/fluentui/docs/src/examples/components/Button/Visual/ButtonExampleComposition.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { Button, useCSS, ButtonProps } from '@fluentui/react-northstar';
+
+type TertieryButtonProps = ButtonProps & {
+  tertiery?: boolean;
+};
+
+const useTertieryButtonStyles = (props, ...rest) => {
```

Comment:
To confirm, is the signature of the `use*Styles()` functions then
```tsx
(props, ...styles) => className
```
Where styles is an array of style objects, style functions taking in theme, or class name strings previously created with `useCSS`?